src/bitmessagekivy/mpybit.py

```diff
@@ -1,33 +1,171 @@
+from kivy.core.window import Window
+
+
+from kivymd.app import MDApp
+from kivymd.uix.label import MDLabel
+from kivymd.uix.list import (
+    OneLineAvatarIconListItem,
+    IRightBodyTouch,
+)
+
+from kivymd.uix.bottomsheet import MDCustomBottomSheet
 
 from pybitmessage.bitmessagekivy.kivy_state import KivyStateVariables
+from pybitmessage.bitmessagekivy.base_navigation import (
+    BaseLanguage, BaseNavigationItem, BaseNavigationDrawerDivider,
+    BaseNavigationDrawerSubheader, BaseContentNavigationDrawer,
+    BaseCustomSpinner
+)
+
+from pybitmessage.bmconfigparser import config
+
+logger = logging.getLogger('default')
+
+data_screen_dict = {}
+
+
+def load_screen_json(data_file="screens_data.json"):
+    """Load screens data from json"""
+
+    with open(os.path.join(os.path.dirname(__file__), data_file)) as read_file:
+        all_data = json.load(read_file)
+        data_screens = list(all_data.keys())
+
+    for key in all_data:
+        if all_data[key]['Import']:
+            import_data = all_data.get(key)['Import']
+            import_to = import_data.split("import")[1].strip()
+            import_from = import_data.split("import")[0].split('from')[1].strip()
+            data_screen_dict[import_to] = importlib.import_module(import_from, import_to)
+    return data_screens, all_data, 'success'
+
+
+def get_identity_list():
+    """Get list of identities and access 'identity_list' variable in .kv file"""
+    identity_list = ListProperty(
+        addr for addr in config.addresses() if config.getboolean(str(addr), 'enabled')
+    )
+    return identity_list
+
+class Lang(BaseLanguage):
+    """UI Language"""
+
+
+class NavigationItem(BaseNavigationItem):
+    """NavigationItem class for kivy Ui"""
+
 
+class NavigationDrawerDivider(BaseNavigationDrawerDivider):
+    """
+    A small full-width divider that can be placed
+    in the :class:`MDNavigationDrawer`
+    """
 
-class NavigateApp(App):
+
+class NavigationDrawerSubheader(BaseNavigationDrawerSubheader):
+    """
+    A subheader for separating content in :class:`MDNavigationDrawer`
+
+    Works well alongside :class:`NavigationDrawerDivider`
+    """
+
+
+class ContentNavigationDrawer(BaseContentNavigationDrawer):
+    """ContentNavigationDrawer class for kivy Uir"""
+
+
+class BadgeText(IRightBodyTouch, MDLabel):
+    """BadgeText class for kivy Ui"""
+
+
+class CustomSpinner(BaseCustomSpinner):
+    """CustomSpinner class for kivy Ui"""
+
+
+class NavigateApp(MDApp):
     """Navigation Layout of class"""
 
     def __init__(self):
         super(NavigateApp, self).__init__()
+        self.data_screens, self.all_data, response = load_screen_json()
         self.kivy_state_obj = KivyStateVariables()
 
-    def build(self):
+    title = "PyBitmessage"
+    identity_list = get_identity_list()
+    image_path = KivyStateVariables().image_dir
+    tr = Lang("en")  # for changing in franch replace en with fr
+
+    def build(self):  # pylint:disable=no-self-use
```

Comment:
why not make this into a static method?

Reply:
`self` is using in this method, i just forgot to remove the `pylint`.